README.md

```diff
@@ -1,3 +1,5 @@
-# ChelFs
+# Chel-FS
 
-A distirbuted filesystem on DAOS object Store.
+## Introduction
+
+Chel-FS is a high performance filesystem built on top of DAOS, a mordern flash medium first storage pool. Chel-FS is aimed for workloads that require high performance filesystem operations. Chel-FS is a layer that is built on top of DAOS. Only filesystem metadata operations will go through Chel-FS, user data will directly flow between DAOS servers and clients. In such a way we can have both high IO bandwidth and IOPS.
```

Comment:
"Chel-FS is aimed for workloads that require high performance filesystem operations."
>>  High performance and Scale.

Reply:
What is Scale. Is it scalability? Users may or may not care about scale. I think we don't add scalability. It is inherited from DAOS.

Reply:
yes ...If there is not scale then POSIX container is enough. Scalability is important here. Specially Compute Scale.